routes/main/homepage.js

```diff
@@ -56,39 +9,21 @@ module.exports = function(config, req, res) {
   }
 
   var options = {
-    appURL: config.appURL,
-    csrf: req.csrfToken(),
-    editorHOST: config.editorHOST,
-    loginURL: config.appURL + "/login",
-    logoutURL: config.logoutURL,
-    queryString: qs,
-    mainURL: env.get("NODE_ENV") === "development" ? "/scripts/main.js" : "/dist/main.js"
+    keepCalmRemixUrl: env.get("KEEP_CALM_REMIX_URL"),
+    keepCalmPublishedUrl: env.get("KEEP_CALM_PUBLISHED_URL"),
+    backToSchoolRemixUrl: env.get("BACK_TO_SCHOOL_REMIX_URL"),
+    backToSchoolPublishedUrl: env.get("BACK_TO_SCHOOL_PUBLISHED_URL"),
+    comicStripRemixUrl: env.get("COMIC_STRIP_REMIX_URL"),
+    comicStripPublishedUrl: env.get("COMIC_STRIP_PUBLISHED_URL"),
+    mainURL: "/homepage/scripts/main.js",
```

Comment:
I'd move this to the HTML, since it's not really a variable that changes--you can hardcode it.